app/feature/feature-login/src/main/kotlin/com/hedvig/android/feature/login/marketing/MarketingDestination.kt

```diff
@@ -171,24 +157,28 @@ private fun MarketingScreen(
               WindowInsets.safeDrawing.only(WindowInsetsSides.Horizontal + WindowInsetsSides.Bottom),
             ),
         ) {
-          HedvigContainedButton(
+          HedvigButton(
             text = stringResource(R.string.SETTINGS_LOGIN_ROW),
             enabled = uiState is MarketingUiState.Success,
             onClick = {
               (uiState as? MarketingUiState.Success)?.run {
                 navigateToLoginScreen(market)
               }
             },
-            modifier = Modifier.testTag("login_button"),
+            modifier = Modifier
+              .fillMaxWidth()
+              .testTag("login_button"),
```

Comment:
This was here for 2 reasons.
When we give instructions to play store on how to login to the app as demo mode, I also gave them information about the resource id of the things they must press
When we tried out some UI tests using Maestro, I needed the items to have a test id in order to tell it what to press.

I am keeping these around since it may be that play store is actually using this